owners/index.js

```diff
@@ -69,3 +75,45 @@ module.exports = app => {
     );
   });
 };
+
+/**
+ * Simple one-day cache for endpoints that do Git commands and file reads.
+ *
+ * TODO(rcebulko): Replace with a real caching solution once the health-check
+ * server gets real functionality.
+ */
+class OneDayCache {
```

Comment:
This made sense when I was writing it, but I'm increasingly thinking it's not necessary.